flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/connectors/hive/HiveSourceBuilder.java

```diff
@@ -91,6 +91,7 @@ public class HiveSourceBuilder {
     private int[] projectedFields;
     private Long limit;
     private List<HiveTablePartition> partitions;
+    private List<String> dynamicPartitionKeys;
```

Comment:
nit:
SInce there exists the concept of static parition /dynamic partition and I think in here it's different from the concept in Hive. The name may be confused. What about `partitionKeysForDynamicFilter` or other name you perfer.